library/src/test/java/com/bumptech/glide/manager/ConnectivityMonitorFactoryTest.java

```diff
@@ -18,7 +18,7 @@
 
   @Before
```

Comment:
Can you rename this DefaultConnectivityMonitorFactoryTest to match the class name?